Gemfile

```diff
@@ -1,2 +0,0 @@
-source 'https://rubygems.org'
```

Comment:
What happened to these 2 files?  

These are the 2 files that tell other people which gems you are using. So when I check out the code it automatically installs the required libs. :)

http://bundler.io/